blog/6-streampark-usercase-joyme.md

```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
 
 ![](/blog/joyme/add_dependency.png)
 
-### **3. 参数配置**
+### **3. Parameter Configuration**
 
-在任务的添加和修改页面中已经罗列了一些常用的参数设置，更多的参数设置则提供了一个 yaml 配置文件，我们这里只是设置了 checkpoint 和 savepoint 这两个配置。一是 checkpoint 的位置，二是 执行 checkpoint 的频率。其他的配置基本没有动，这部分用户可以根据自己的需要按需配置。
+The task addition and modification page has listed some common parameter settings, and for more extensive configuration options, a yaml configuration file is provided. Here, we've only set up checkpoint and savepoint configurations. The first is the location of the checkpoint, and the second is the frequency of checkpoint execution. We haven't changed other configurations much; users can customize these settings as per their needs.
 
-剩下的一些参数设置就要根据作业的具体情况去对症下药的配置了，处理的数据量大了，逻辑复杂了，可能就需要更多的内存，并行度给多一些。有时候需要根据作业的运行情况进行多次调整。
+The rest of the parameter settings should be configured according to the specific circumstances of the job. If the volume of data processed is large or the logic is complex, it might require more memory and higher parallelism. Sometimes, adjustments need to be made multiple times based on the job's operational performance.
 
 ![](/blog/joyme/checkpoint_configuration.png)
 
-### **4. 动态参数设置**
+### **4. Dynamic Parameter Settings**
 
-由于我们的模式部署是 on Yarn，在动态选项配置里配置了 Yarn 的队列名称。也有一些配置了开启增量的 Checkpoint 选项和状态过期时间，基本的这些参数都可以从 Flink 的官网去查询到。之前有一些作业确实经常出现内存溢出的问题，加上增量参数和过期参数以后，作业的运行情况好多了。还有就是 Flink Sql 作业设计到状态这种比较大和逻辑复杂的情况下，我个人感觉还是用 Streaming 代码来实现比较好控制一些。
+Since our deployment mode is on Yarn, we configured the name of the Yarn queue in the dynamic options. Some configurations have also been set to enable incremental checkpoints and state TTL (time-to-live), all of which can be found on Flink's official website. Before, some jobs frequently encountered out-of-memory issues. After adding the incremental parameter and TTL, the job operation improved significantly. Also, in cases where the Flink SQL job involves larger states and complex logic, I personally feel that it's better to implement them through streaming code for more control.
 
-- -Dyarn.application.queue= yarn队列名称
+- -Dyarn.application.queue= [Yarn queue name]
 - -Dstate.backend.incremental=true
-- -Dtable.exec.state.ttl=过期时间
+- -Dtable.exec.state.ttl= [expiration time]
 
-完成配置以后提交，然后在 application 界面进行部署。
+After completing the configuration, submit it, and then deploy it from the application interface.
```

Comment:
After completing the configuration, submit & deploy it from the application interface.